specs/commandline/policy.md

```diff
@@ -0,0 +1,696 @@
+  -n, --name        string      name of the trust policy statement
+      --reference   string      reference to the artifact
+      --scope       string      repository URI, e.g. localhost:5000/namespace/repo_name
+```
+
+### notation policy update
+
+```text
+Update the existing trust policies.
+
+Usage:
+  notation policy update (--file <policy_file_path> | --name <policy_name>) [flags]
+
+Flags:
+      --custom-level       stringArray   {key}={value} pairs that represent a custom level to override existing verification level
+  -f  --file               string        path to a trust policy file in JSON format
+  -n  --name               string        name of the trust policy statement
+  -h, --help               help for add
+      --scope              stringArray   repository URIs to which the policy applies (default ["*"])
+      --trust-store        stringArray   trust stores in format "<trust_store_type>:<trust_store_name>"
+      --verification-level string        verification level, options: "strict", "permissive", "audit", "skip" (default "strict")
+      --x509-cert          stringArray   paths to x509 certificate file that certificate subject is retrieved from
+      --x509-id            stringArray   trust identities, user trusted x509 certificate subjects
+```
+
+## Usage
+
+### Add a trust policy by configuring the properties from command line
+
+The following table shows how to configure properties for a trust policy and default values.
+
+| Property Name       | Necessity | flag to configure                    | Default value |
+| ------------------- | --------- | ------------------------------------ | ------------- |
+| `name`              | Required  | `--name`                             | N/A           |
+| `trustStores`       | Required  | `--trust-store`                      | N/A           |
+| `registryScopes`    | Optional  | `--scope`                            | `["*"]`       |
+| `level`             | Optional  | `--verification-level`               | `"strict"`    |
+| `override`          | Optional  | `--custom-level`                     | nil           |
+| `trustedIdentities` | Optional  | `--x509-id` or `--x509-cert` or both | `["*"]`       |
+
+```shell  
+notation policy add --name wabbit-network-dev --scope "dev.wabbitnetworks.io/net-monitor" --trust-store "ca:wabbit-network-dev" --x509-id "C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io" --verification-level "strict"
+```
+
+The execution of `add` fails in any of below cases:
+
+- The trust policy name exists.
+- More than one trust policy that uses a global scope, that is, the value of `registryScopes` is `["*"]`.
+- The values of `--x509-id` or `--x509cert` overlap. For example, the following two identity values are overlapping:
+  - "C=US, ST=WA, O=wabbit-network.io, OU=org1"
+  - "C=US, ST=WA, O=wabbit-network.io"
+
+Upon successful execution, the added trust policy is printed out. For example:
+
+In text format
```

Comment:
Do we really need text format for the `notation policy` command set? To be honest, the text format is less readable than the json format for a human.